lib/metacat.properties

```diff
@@ -93,6 +93,8 @@ storage.hashstore.defaultNamespace=https://ns.dataone.org/service/types/v2.0#Sys
 storage.hashstore.fileNameAlgorithm=SHA-256
 storage.hashstore.directory.width=2
 storage.hashstore.directory.depth=3
+# The time out for waiting the completness of the conversion jobs. The unit is day.
+storage.hashstore.converterTimeout=5
```

Comment:
not critical - but it's usually nice to include the units in the name, so there's no chance of confusion (e.g. `storage.hashstore.converterTimeoutDays`)